### PR TITLE
Expose a method **generateOutputFile** so it can be called from both sfdx plugin & VsCode Extension

### PR DESCRIPTION
**Pull request by nvuillam**

### CODE REVIEW

**Review by RubenHalman — COMMENTED** [review]
Thank you so much for this. LGTM, will you also create the pr for this in the sfdx app simultaneously or how would you like to proceed?

**Comment by nvuillam** [comment]
@RubenHalman i'll be able to call it from sfdx & VsCode once scanner-core will be released, to be able to upgrade the dependency then call the method :)

**Comment by RubenHalman** [comment]
@nvuillam Roger that. Please allow me a few days to verify the compatibility/compilation(there should be no issue and this is a small task, but I don't have this automated yet). Meanwhile if you want to work on the sf plugin already, you can import/test local modules by replacing the import statement to point to a module you compile locally, like so: 

Replace: 
`import { RuleResult } from "lightning-flow-scanner-core/out/main/models/RuleResult";
`
With:
`import { RuleResult } from "/Users/rubenhalman/Projects/lightning-flow-scanner-core/lib";`

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] (master branch content)
+
+- Add your updates here :)
+
+## [v2.27.0] - 2023-12-16
+
+- Expose a method **generateOutputFile** so it can be called from both sfdx plugin & VsCode Extension
```
Comment (RubenHalman) [inline CHANGELOG.md:13]: @nvuillam Would you be so kind to drop a few lines on the CSV / XLSX functionality features? From what I gather from your pr, it is probably helpful to document the fields exposed: ["Flow API Name", "Rule Name", "Item", "Severity", "Rule description", "Rule Type", "Flow Type", "Flow Label"]. Do you think you can help me write something that I can use as a baseline of instruction for the readme?

**Review by RubenHalman — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nvuillam) [reply CHANGELOG.md:13]: @RubenHalman these are the exact same fields that are output in the console, naming them is probably enough for understanding :)

**Comment by RubenHalman** [comment]
this pr needs to be rebased on 2.30

**Pull request closed without merging**